static/frontEndLayout.js

```diff
@@ -608,4 +619,17 @@ var tick = function() {
         }
     };
     document.getElementById("title").innerHTML = readyToHarvest + " plots ready - Garden Sim 2K17";
+
+    //move out of tick to separate function, called on every game state change
+    for (var recipeId in RECIPE_CONFIG) {
+        for (var resource in RECIPE_CONFIG[recipeId].cost) {
+            if (state.resources[resource] < RECIPE_CONFIG[recipeId].cost[resource]) {
```

Comment:
rather than changing the style, swapping out the class leaves the layout data in your css file.

you also might consider getting rid of the `cursor: pointer` when disabling so it doesn't look clickable, but that's a style preference